### ISSUE TITLE
[plugins/unocss] fix `presetIcons` significantly slows down hydration

### ISSUE THREAD

**Issue by miguelrk** [netzo/netzo#172]
**TLDR**: enabling CSR mode will significantly slow-down hydration due primarily to the `presetIcons` preset included by the `presetShadn` preset. This is probably due to the `presetShadn` including the entire MDI icons collection, which is then base64-encoded to be loaded as an ESM import in the plugin's entrypoint script which initializes the unocss runtime.

The current workaround is using the `"safelist"` property in `presetShadcn` to force-include all dynamically injected classes for components which mount/unmount dynamically (e.g. dialogs, sheets). Additional dynamic classes can always be included in the `"safelist"` of the `uno.config` file as well. This way, `netzo/components` can be used without having to enable CSR mode for the `netzo.unocss` plugin, thus avoiding slowing down hydration.

The `presetIcons` is desired, so this workaround (disabling CSR mode and using `"safelist"`) is preffered over not using icons.
